src/scripts/tests/simpleKingMoves.spec.ts

```
@@ -0,0 +1,32 @@
+
+    assert.lengthOf(moves!, 3);
+  });
+
+  test('Long moves', () => {
+    const board = new Board();
+    const piece = new Piece(false, new Vector2(0, 0), true);
+    board.addPiece(piece);
+
+    const moves = MoveCalculator.calculateMoves(board, piece);
+
+    assert.lengthOf(moves!, GameConstants.CellsPerSide * 3 - 3);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)